pkgs/by-name/li/libre-graph-api-cpp-qt-client/package.nix

```diff
@@ -19,15 +18,14 @@ stdenv.mkDerivation rec {
 
   sourceRoot = "${src.name}/client";
 
-  nativeBuildInputs = [ cmake wrapQtAppsHook ];
-  buildInputs = [ qtbase ];
-
-  cmakeFlags = [  ];
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ qt6.qtbase ];
+  dontWrapQtApps = true;
 
   meta = with lib; {
     description = "C++ Qt API for Libre Graph, a free API for cloud collaboration inspired by the MS Graph API";
     homepage = "https://owncloud.org";
-    maintainers = with maintainers; [ qknight hellwolf ];
+    maintainers = with maintainers; [ hellwolf ];
```

Comment:
Why remove qknight?

Reply:
This is the new package required by owncound after I started to do more packaging. @qknight is still the maintainer of owncloud.

In fact these 3 packages are so tightly coupled that, at the moment it doesn't make a difference since qknight is still in the maintainer list of owncloud-client package itself.

Reply:
checked the `git blame`, looks good